crates/solidity/testing/perf/npm/package-lock.json

```diff
@@ -0,0 +1,2001 @@
+{
```

Comment:
since `@slang-private/solidity-testing-perf-npm` is part of the root workspace, I wonder if this lock file is extra?